addons/account/i18n/account.pot

```diff
@@ -12048,12 +12073,19 @@ msgstr ""
 
 #. module: account
 #: model:ir.model.fields,help:account.field_account_payment__currency_id
-#: model:ir.model.fields,help:account.field_account_payment__destination_account_id
 #: model:ir.model.fields,help:account.field_account_payment_register__currency_id
 #: model:ir.model.fields,help:account.field_account_payment_register__source_currency_id
 msgid "The payment's currency."
 msgstr ""
 
+#. module: account
+#: code:addons/account/models/account_move.py:0
+#, python-format
+msgid ""
+"The recipient bank account link to this invoice is archived.\n"
+"So you cannot confirm the invoice."
+msgstr ""
+
```

Comment:
Here, there is my part of code that needs to be translated. But I downloaded the `.pot` from the runbot, and added it here. But I don't know if this is the way to do it? Because there is a quite big diff.

Reply:
No you just have to add the new entry in the po file

Reply:
Ok, thanks